src/Map/doc/index.rst

```diff
@@ -0,0 +1,285 @@
+Symfony UX Map
+==============
+
+**EXPERIMENTAL** This component is currently experimental and is likely
+to change, or even change drastically.
+
+Symfony UX Map is a Symfony bundle integrating interactive Maps in
+Symfony applications. It is part of `the Symfony UX initiative`_.
+
+Installation
+------------
+
+.. caution::
+
+    Before you start, make sure you have `StimulusBundle configured in your app`_.
+
+Install the bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-map
+
+If you're using WebpackEncore, install your assets and restart Encore (not
+needed if you're using AssetMapper):
+
+.. code-block:: terminal
+
+    $ npm install --force
+    $ npm run watch
+
+    # or use yarn
+    $ yarn install --force
+    $ yarn watch
+
+After installing the bundle, ensure the line ``{{ ux_map_script_tags() }}`` is present in your Twig template, e.g.:
+
+.. code-block:: twig
+
+        {% block javascripts %}
+            {% block importmap %}{{ importmap('app') }}{% endblock %}
+            {{ ux_map_script_tags() }}
+        {% endblock %}
```

Comment:
Can we put all this in a dedicated "WebpackEncore" part (at the end of the doc) ?

Reply:
Yes we can, except for `ux_map_script_tags()`